docs-website-src/content/blockers.md

```diff
@@ -106,15 +106,15 @@ We block if your machine has multiple network interface cards (NICs) using kerne
   * The "default" approach in `network-scripts` config files of specifying NICs by `DEVICE` can cause issues due to the above.
   * To find a more in-depth explanation of *why* this is a problem (and what to do about it), read Freedesktop.org's <a href="https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/" target="_blank">Predictable Network Interface Names</a> documentation.
 
-One way to prevent these issues is to assign a custom name in the configuration and re-initialize NICs ahead of time.
+On systems configured in this manner, we will offer to automatically assign a custom names (from ethX to cpethX) in the configuration and re-initialize NICs ahead of elevation.
 
 ### MySQL version
 
 A MySQL upgrade **cannot be** in progress.
 
 If the version of MySQL/MariaDB installed on the system is not supported on RHEL 8-based distributions, you **must** upgrade to a supported version. If cPanel manages the MySQL installation, we will offer to upgrade MySQL automatically to MariaDB 10.6 during elevation.
 
-The system **must** not be set up to use a remote database server.
+The system **must** not be set up to use a remote database server. In cases where the system is set up to use a remote server, a local server will be temporarily configured and enabled for the duration of the elevation, and the remote instance will be re-enabled once the elevation completes.
```

Comment:
We only do this if the local version of MySQL will be unsupported after the elevation is complete:

```
    # If the database version is supported on the new OS version, then no need to upgrade
    return if Elevate::Database::is_database_version_supported( Elevate::Database::get_local_database_version() );

    $self->_ensure_localhost_mysql_profile_is_active(1);
```

SUGGESTION:

```
### MySQL version

A MySQL upgrade **cannot be** in progress.

If the local version of MySQL/MariaDB installed on the system is not supported on RHEL 8-based distributions, you **must** upgrade to a supported version. If cPanel manages the MySQL installation, we will offer to upgrade MySQL automatically to MariaDB 10.6 during elevation.

**NOTE:** In cases where an upgrade is necessary **and** the system is set up to use a remote server, a local server will be temporarily configured and enabled for the duration of the elevation, and the remote instance will be re-enabled once the elevation completes.
```